backend/src/main/java/io/github/projektalmanac/amoxtli/backend/Application.java

```diff
@@ -27,6 +36,48 @@ public void postConstruct() {
 		user.setId(1);
 		user.setVerifiedEmail(true);
 
+		var user2 = new User();
+		user2.setName("Carlos");
+		user2.setLastName("Castro");
+		user2.setId(2);
+		user2.setVerifiedEmail(true);
+
+		var user3 = new User();
+		user3.setName("Samuel");
+		user3.setLastName("Morales");
+		user3.setId(3);
+		user3.setVerifiedEmail(true);
+		List<Book> libro = new ArrayList<>();
+		List<Book> libro2 = new ArrayList<>();
```

Comment:
```suggestion
		List<Book> libros = new ArrayList<>();
		List<Book> libros2 = new ArrayList<>();
```

Son arreglos, sus nombres deben ir en pluarl